packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -41,10 +42,14 @@ const NavLogo = ({
   link,
   ...customProps
 }) => {
-  const image = (link) ?
-    <SmartLink {...link}> <img className={cx('img')} src={url} alt={altText} /> </SmartLink> :
-    <img className={cx('img')} src={url} alt={altText} />;
-  const body = (isCard && !!url) ? <Card.Body> {image} </Card.Body> : image;
+  const imgProps = {
+    className: cx('img'),
+    ...url && { src: url },
+  };
+
+  const imgTag = (<img alt={altText} {...imgProps} />);
+  const imgContent = (link) ? <SmartLink {...link}>{imgTag} </SmartLink> : imgTag;
+  const body = (isCard && !!url) ? <Card.Body> {imgContent} </Card.Body> : imgContent;
```

Comment:
is the `!!url` double negative some crazy javascript truthy hack?

Reply:
as @WhiteLight1107 stated above, this is a truthify statement. any value that is not: false, 0, "", null, undefined, or NaN, will return true, otherwise false. https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
I love the name "Thruthy"